sql/core/src/test/resources/sql-tests/results/postgreSQL/text.sql-jdk17.out

```diff
@@ -0,0 +1,360 @@
+cannot resolve 'concat_ws(CAST(NULL AS STRING), 10, 20, NULL, 30)' due to data type mismatch: argument 2 requires (array<string> or string) type, however, '10' is of int type. argument 3 requires (array<string> or string) type, however, '20' is of int type. argument 4 requires (array<string> or string) type, however, 'NULL' is of void type. argument 5 requires (array<string> or string) type, however, '30' is of int type.; line 1 pos 7
+
+
+-- !query
+select reverse('abcde')
+-- !query schema
+struct<reverse(abcde):string>
+-- !query output
+edcba
+
+
+-- !query
+select i, left('ahoj', i), right('ahoj', i) from range(-5, 6) t(i) order by i
+-- !query schema
+struct<>
+-- !query output
+org.apache.spark.sql.AnalysisException
+cannot resolve 'substring('ahoj', 1, t.i)' due to data type mismatch: argument 3 requires int type, however, 't.i' is of bigint type.; line 1 pos 10
+
+
+-- !query
+/*
+ * format
+ */
+select format_string(NULL)
+-- !query schema
+struct<format_string(NULL):string>
+-- !query output
+NULL
+
+
+-- !query
+select format_string('Hello')
+-- !query schema
+struct<format_string(Hello):string>
+-- !query output
+Hello
+
+
+-- !query
+select format_string('Hello %s', 'World')
+-- !query schema
+struct<format_string(Hello %s, World):string>
+-- !query output
+Hello World
+
+
+-- !query
+select format_string('Hello %%')
+-- !query schema
+struct<format_string(Hello %%):string>
+-- !query output
+Hello %
+
+
+-- !query
+select format_string('Hello %%%%')
+-- !query schema
+struct<format_string(Hello %%%%):string>
+-- !query output
+Hello %%
+
+
+-- !query
+select format_string('Hello %s %s', 'World')
+-- !query schema
+struct<>
+-- !query output
+java.util.MissingFormatArgumentException
+Format specifier '%s'
+
+
+-- !query
+select format_string('Hello %s')
+-- !query schema
+struct<>
+-- !query output
+java.util.MissingFormatArgumentException
+Format specifier '%s'
+
+
+-- !query
+select format_string('Hello %x', 20)
+-- !query schema
+struct<format_string(Hello %x, 20):string>
+-- !query output
+Hello 14
+
+
+-- !query
+select format_string('%1$s %3$s', 1, 2, 3)
+-- !query schema
+struct<format_string(%1$s %3$s, 1, 2, 3):string>
+-- !query output
+1 3
+
+
+-- !query
+select format_string('%1$s %12$s', 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12)
+-- !query schema
+struct<format_string(%1$s %12$s, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12):string>
+-- !query output
+1 12
+
+
+-- !query
+select format_string('%1$s %4$s', 1, 2, 3)
+-- !query schema
+struct<>
+-- !query output
+java.util.MissingFormatArgumentException
+Format specifier '%4$s'
+
+
+-- !query
+select format_string('%1$s %13$s', 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12)
+-- !query schema
+struct<>
+-- !query output
+java.util.MissingFormatArgumentException
+Format specifier '%13$s'
+
+
+-- !query
+select format_string('%0$s', 'Hello')
+-- !query schema
+struct<>
+-- !query output
+java.util.IllegalFormatArgumentIndexException
+Illegal format argument index = 0
+
```

Comment:
```
--PostgreSQL throw ERROR:  format specifies argument 0, but arguments are numbered from 1
select format_string('%0$s', 'Hello');
```
The SQL in `postgreSQL/text.sql` has this different behavior between Java8 and Java17, but it seems that the behavior of Java 17 is expected, so do Independent verification here.